### PR TITLE
Note more tools for HDF5 files

### LINKED ISSUE #1186: Add H5Web to list of tools to view NeXus files
Note [H5Web](https://h5web.panosc.eu), a new extension for [Microsoft VisualStudio Code Editor](https://code.visualstudio.com/) to browse and visualize HDF5 files

Add it to this page and section:
https://manual.nexusformat.org/utilities.html#hdf-tools

### LINKED ISSUE #1187: Add tiled to list of tools to view NeXus files
Note [tiled](https://blueskyproject.io/tiled/), a _data access service_ for data-aware portals and data science tools, provides a way to browse and visualize HDF5 files.

Add it to this page and section:
https://manual.nexusformat.org/utilities.html#hdf-tools

### PR DESCRIPTION
**Pull request by prjemian**
- close #1186
- close #1187

### CODE REVIEW

File: manual/source/utilities.rst [hunk 1]
```diff
@@ -349,32 +349,56 @@ as a data format.  It is not intended to be a complete list of all available pac
...
-APIs for many programming languages that will allow 
+APIs for many programming languages that will allow
 low level programmatic access to the data structures.
 
-.. index:: 
+.. index::
+	h5wasm; tools
+
+**h5wasm** (https://h5web.panosc.eu/h5wasm)
+    Web site allowing one to view HDF5 content from a URL.
```
Comment (PeterC-DLS) [inline manual/source/utilities.rst:361]: Demonstration web site of self-contained Webassembly HDF5 reader and writer

File: manual/source/utilities.rst [hunk 2]
```diff
@@ -349,32 +349,56 @@ as a data format.  It is not intended to be a complete list of all available pac
...
+
+    Example from the NeXus manual: ``simple_example_basic.nexus.hdf5``
+
+    https://h5web.panosc.eu/h5wasm?url=https%3A%2F%2Fmanual.nexusformat.org%2F_downloads%2Fb9b76d34af0f606d0e0958a234054608%2Fsimple_example_basic.nexus.hdf5
+
+.. index::
+	H5Web; tools
+
+**H5Web** (https://h5web.panosc.eu/)
```
Comment (PeterC-DLS) [inline manual/source/utilities.rst:370]: vscode-h5web https://marketplace.visualstudio.com/items?itemName=h5web.vscode-h5web
(https://github.com/silx-kit/vscode-h5web)

**Comment by woutdenolf** [comment]
Comment by @axelboc (h5web developer). Ping @loichuder

> hmmm it's a bit weird
> 
> h5wasm is the WebAssembly port of the HDF5 C Library, but then it points to a demo of H5Web
> 
> I would start by mentioning H5Web as a "toolkit for exploring and visualising HDF5 files"
> 
> with maybe a paragraph explaining that it can work with various back-end solutions for serving HDF5 files, and then pointing to >H5Grove, HSDS from there
> 
> it can also work with h5wasm for visualising local HDF5 files

**Comment by prjemian** [comment]
Looks *weird* because the list was alphabetized.  But here, there is some hierarchy to be captured.

File: manual/source/utilities.rst [hunk 3]
```diff
@@ -349,32 +349,67 @@ as a data format.  It is not intended to be a complete list of all available pac
...
+**h5wasm** (https://github.com/usnistgov/h5wasm):
+
+    A `WebAssembly <https://webassembly.org/>`__
+    port of the HDF5 C library, which allows reading and writing
+    HDF5 files from JavaScript (i.e. no need for a back-end server at all).
 
-.. index:: 
+.. index::
+    jupyterlab-h5web; tools
+    vscode-w5web; tools
```
Comment (loichuder) [inline manual/source/utilities.rst:367]:
```suggestion
    vscode-h5web; tools
```

**Review by loichuder — APPROVED** [review]

**Pull request merged**